src/front-end/src/components/MenteeFormAndEmbedder/MenteeFormAndEmbedder.test.js

```diff
@@ -41,3 +41,32 @@ test('renders the correct header2 text', () => {
     'The form below is an example of the mentee form:',
   );
 });
+
+function isInIframe() {
+  return window.top !== window.self;
+}
+
+// Referenced from: https://theunlikelydeveloper.com/jest-mock-window-test-iframes/
+describe('isInIframe', () => {
+  it('should return true if in iframe', () => {
+    // Extract the real properties so we can put them back after this test
+    const { top, self } = window;
+
+    // Delete the real properties from window so we can mock them
+    delete window.top;
+    delete window.self;
+
+    // Mock window.top and window.self as different objects (we don't
+    // need anything else from the original objects here, so {} is fine)
+    window.top = {};
+    window.self = {};
+
+    // Test the code
+    const result = isInIframe();
+    expect(result).toBe(true);
+
+    // Restore the original values so other tests will function normally
+    window.top = top;
+    window.self = self;
+  });
+});
```

Comment:
I copy pasted this in MenteeDisplay.test.js and it passed, which means it doesn't test the iFrame. What we need to do is:
1. Render component
2. [Simulate user click on "Generate iFrame" button](https://testing-library.com/docs/dom-testing-library/api-events/)
3. Check that the iframe is indeed open. You may run into an issue because opening the iFrame makes a call to firebase, but if we get there we will deal with it.

Reply:
Okay, I will try to do this soon and push the new commit!

Reply:
I tried to work on the test for 45mins and I cant seem to get it to work. I pushed the test that seemed the most right but Im not too sure how to go further than this point. I think at this point it is better to focus on the final documents instead of trying to solve the unit tests (unless it is an easy fix).